_includes/head.html

```diff
@@ -16,6 +16,10 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="/stylesheets/bootstrap.css">
     
+	<!-- X3D styles and scripts -->
+	<link rel='stylesheet' type='text/css' href='http://www.x3dom.org/download/x3dom.css'></link> 
```

Comment:
For reference:

It's not allowed in html5 for link: neither </link> and not even <link some content/>
It's mandatory in xhtml, so in xhtml <link some content />

So for pure html: it should be <link ...>
For xhtml <link .../>

The browsers ignore / the at the <link/>.

I will follow pure html5 then.